Flatiron - Tuples chapter.md

```diff
@@ -0,0 +1,115 @@
+
+##What's a Tuple?
+Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
+
+**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
+```
+// Array
+let list = [ "Darth", "Ben", "Luke" ]
+print( list[1] )
+```
+Arrays are pretty useful but they're somewhat limited in that their contents can *only* be accessed via indices. 
+
```

Comment:
In general, if we were to present the student with a problem with regards to something they DID know and try to relate it to something we're currently teaching them (in this case `tuples`), we like to tie the two things in immediately and answer the why.

Here, you presented a problem which is great! But, if you are to present a problem/situation like this, it's good to just hit the student immediately with what it is that tuples do to solve this problem! The separation of that will leave a sentence like this to not be something the student might remember or take away from this lesson.

Maybe writing something like this:

Arrays are useful in that they are a collection that holds information for us in an ordered fashion. But what if we have specific data to an individual that we don't want to store in one place. I want to store my height and weight somewhere, I can do so using arrays
```swift
let attributes = [185, 70]
```

But... how do I know which one represents my height, and one which one represents my weight? This doesn't seem like the best solution. We can solve this problem using a `tuple` (ideally this problem would be solved using a `Class`, but that is something we will cover in a future lesson).

```swift
let jim = (weight: 185, height: 70)
```

Here, we're introducing a `tuple` which is now storing two pieces of information that are named which makes it easier for the user of this newly made `jim` variable.